jetbrains-ultimate/tst/software/aws/toolkits/jetbrains/services/lambda/go/GoLambdaHandlerResolverTest.kt

```diff
@@ -335,6 +350,8 @@ class GoLambdaHandlerResolverTest {
             if (shouldBeFound) {
                 assertThat(lambdas).hasSize(1)
                 assertThat(lambdas[0]).isInstanceOf(GoFunctionDeclaration::class.java)
+                // assert that the lambda was found in a temp directory (i.e. from the injected handler and not something from the go library)
+                assertThat(lambdas[0].containingFile.containingDirectory.isPhysical).isFalse
```

Comment:
Should we check it is under a content root to be more accurate of an assert? the isPhyiscal==false is kind of confusing

Reply:
moved to integ to actually test this properly